ci/generate-docs.sh

```diff
@@ -1,51 +1,19 @@
 #!/bin/bash
 
-# called by Travis CI
-
 set -ex
 
-if [[ "false" != "$TRAVIS_PULL_REQUEST" ]]; then
-	echo "Not deploying pull requests."
-	exit
-fi
-
-if [[ "$TRAVIS_BRANCH" != "$DEPLOY_BRANCH" ]]; then
-	echo "Not on the '${DEPLOY_BRANCH}' branch."
-	exit
-fi
-
 # Path to the documentation directory
-VIP_DOCS_DIR="${TRAVIS_BUILD_DIR}/phpdoc"
+VIP_DOCS_DIR="phpdoc"
 # Path to the PHPDoc application
 VIP_PHPDOC_DIR="/tmp/phpdoc"
 
-# Get the encrypted private key from the repo settings
-# This is the private pair to the "Travis GH Pages Deploy Key"
-# The public key portion can be deleted here:
-# https://github.com/Automattic/vip-go-mu-plugins/settings/keys
-# The private key is in Travis settings here:
-# https://travis-ci.org/Automattic/vip-go-mu-plugins/settings
-# Before pasting the key into the setting, I replaced newlines
-# with \n and surrounded it with double quotes, e.g. "KEY\nHERE\n".
-
-# Turn off echo for the private key
-set +x
-echo -e $VIP_GITHUB_DEPLOY_KEY > /tmp/vip_deploy_key
-chmod 600 /tmp/vip_deploy_key
-set -x
-
-# Ensure we use our deploy key when connecting to GitHub,
-# this allows us to write (as the deploy key has write perms)
-echo -e "\nHost github.com \n  IdentityFile /tmp/vip_deploy_key \n" >> ~/.ssh/config
-
-cat ${TRAVIS_BUILD_DIR}/ci/known_hosts ~/.ssh/known_hosts
-cp ${TRAVIS_BUILD_DIR}/ci/known_hosts ~/.ssh/known_hosts
+cp ./ci/known_hosts ~/.ssh/known_hosts
 
 # Clone the repo at the `gh-pages` branch, getting only the commits for that branch
-git clone --branch gh-pages --single-branch "git@github.com:${TRAVIS_REPO_SLUG}.git" ${VIP_DOCS_DIR}
+git clone --branch gh-pages --single-branch "git@github.com:Automattic/vip-go-mu-plugins.git" ${VIP_DOCS_DIR}
 
-# Composer runs faster without Xdebug, and we don't need Xdebug any more
-phpenv config-rm xdebug.ini
+# # Composer runs faster without Xdebug, and we don't need Xdebug any more
+# phpenv config-rm xdebug.ini
```

Comment:
Is there any reason to comment this line?

Reply:
Just that the php docker image doesn't come with `phpenv` and it runs fast enough without this optimization so I felt, that it is simpler to just drop this. I did forget to remove the commented code though, it is removed now..